test/sync.js

```diff
@@ -71,6 +71,22 @@ test('an additional salt can be provided', t => {
   t.true(actual === expected)
 })
 
+test('salt can be a JSON object', t => {
```

Comment:
Ah true. There's no test for string values but meh that would just be for completeness sake.